### PR TITLE
add notification manager introduction

### PR DESCRIPTION
**Pull request by wanjunlei**
add notification manager introduction

### CODE REVIEW

File: content/en/notification-manager/notification-manager.md [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+
+Supported senders includes:
+- Prometheus Alertmanager
+- Custom sender (Coming soon)
+
+Supported receivers includes:
+- Email
+- [Wechat Work](https://work.weixin.qq.com/)
+- Slack 
+- Webhook (Coming soon)
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:17]: @wanjunlei  Will webhook be supported in v3.0?

File: content/en/notification-manager/notification-manager.md [hunk 2]
```diff
@@ -0,0 +1,404 @@
+---
+title: "Notification Manager"
+keywords: "kubernetes, kubesphere, notification manager, email, wechat, slack"
+description: "Multi-tenant notification manager on KubeSphere"
+---
+
+Notification Manager manages notifications in kubesphere. It receives alerts or notifications from different senders and then send notifications to various tenant receivers based on alerts/notifications' tenant label like "namespace".
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:7]: `various tenant` => `multi-tenant` ?

File: content/en/notification-manager/notification-manager.md [hunk 3]
```diff
@@ -0,0 +1,404 @@
...
+- Slack 
+- Webhook (Coming soon)
+
+## QuickStart
+
+### Enable Notification Manager before Installation
+
+> Note: This guide is only used for installing KubeSphere on Linux machines. If you are going to install KubeSphere and Notification Manager on your own Kubernetes cluster, please see [ks-installer](https://github.com/kubesphere/ks-installer).
+
+Before starting the installation, you need to change the value of `monitoring.notification-manager.enabled` to `true` in `conf/common.yaml` as follows to enable notification manager, then you can go back to [All-in-One](../all-in-one) or [Multi-Node](../multi-node) guide to continue your installation.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:25]: `to enable notification manager` is redundant, remove it.

File: content/en/notification-manager/notification-manager.md [hunk 4]
```diff
@@ -0,0 +1,404 @@
...
+
+Then set Notification Manager from `False` to `True`.
+
+```yaml
+    monitoring:
+      notification:
+        enabled: true
+```
+
+Save it and exit. Notification Manager will be installed automatically for you. You can inspect the logs of ks-installer Pod to [verify the installation status](../verify-components), and wait for the successful result logs output.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:51]: remove `for you`

File: content/en/notification-manager/notification-manager.md (hunk 1 above)
Comment (wanjunlei) [reply content/en/notification-manager/notification-manager.md:17]: no

File: content/en/notification-manager/notification-manager.md [hunk 5]
```diff
@@ -0,0 +1,404 @@
...
+
+### Config Prometheus Alertmanager to send alerts to Notification Manager
+
+Notification Manager use port `19093` and API path `/api/v2/alerts` to receive alerts sending from Prometheus Alertmanager of kubsphere.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:55]: use=>uses

Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:55]: use=>uses, kubsphere=>KubeSphere

File: content/en/notification-manager/notification-manager.md [hunk 6]
```diff
@@ -0,0 +1,404 @@
...
+To receive Alertmanager alerts, eidt secret alertmanager-main in kubesphere-monitoring-system, add webhook config like below to the `receivers` section of alertmanager.yaml:
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:56]: edit the Secret `alertmanager-main` in the namespace `kubesphere-monitoring-system`

File: content/en/notification-manager/notification-manager.md [hunk 7]
```diff
@@ -0,0 +1,404 @@
...
+       - "url": "http://notificatio-nmanager-svc.kubesphere-monitoring-system.svc:19093/api/v2/alerts"
+```
+
+### Config receivers
+
+Notification Manager now supports three type receivers, email, Wechat Work, slack. Only administrator can config receivers.
+
+#### Email
+
+If a tenant named test-user want to receive notification with email, it must be create a email receiver like this.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:71]: test-user should be bold like: **test-user**.

File: content/en/notification-manager/notification-manager.md [hunk 8]
```diff
@@ -0,0 +1,404 @@
...
+metadata:
+  labels:
+    app: notification-manager
+  name: test-user-secret
+  namespace: kubesphere-monitoring-system
+type: Opaque
+EOF
+```
+
+The emailConfigSelector is a selector to selecte EmailConfig for email receiver, if the emailConfigSelector is nil, receiver will use default email config. You can create a default email config like this.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:129]: use default email config. => use the default email config.

File: content/en/notification-manager/notification-manager.md [hunk 9]
```diff
@@ -0,0 +1,404 @@
...
+kind: Secret
+metadata:
+  labels:
+    app: notification-manager
+  name: default-email-secret
+  namespace: kubesphere-monitoring-system
+type: Opaque
+```
+
+Tanten email receiver only receive notification happend in the namespace which tanten in. If you want to receive notification without namespace, you can create a global email receiver like this.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:166]: What is the `Tanten` mean?

File: content/en/notification-manager/notification-manager.md (hunk 7 above)
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:71]: it must be create a email receiver like this.=> you need to

File: content/en/notification-manager/notification-manager.md [hunk 10]
```diff
@@ -0,0 +1,404 @@
...
+  - global@xyz.com
+EOF
+```
+
+>Global email receiver use default email config.
+
+#### Wechat Work
+
+Notification Manager supports sending notification to Wechat Work. 
+If a tenant named test-user want to receive notification with Wechat Work, it must be create a wechat receiver like this.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:188]: If a tenant named test-user want to => If a tenant named test-user who wants to

File: content/en/notification-manager/notification-manager.md (hunk 7 above)
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:71]: receive notification with email => `receive a notification from email` might be exact.

File: content/en/notification-manager/notification-manager.md (hunk 10 above)
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:188]: receive notification with Wechat Work => `receive a notification from Wechat Work` might be exact.

File: content/en/notification-manager/notification-manager.md [hunk 11]
```diff
@@ -0,0 +1,404 @@
...
+    app: notification-manager
+  name: test-user-secret
+  namespace: kubesphere-monitoring-system
+type: Opaque
+EOF
+```
+
+>WechatApiCorpId is the id of your Wechat Work. WechatApiAgentId is the id of app which sending message to user in your Wechat Work, wechatApiSecret is the secret of this app, you can get these two parameters in App Managerment of your Wechat Work. Note that any user, party or tag who wants to rerceive notifications must be in the allowed users list of this app.
+
+The wechatConfigSelector is a selector to selecte WechatConfig for wechat receiver, if the wechatConfigSelector is nil, wechat receiver will use default wechat config. You can create a default wechat config like this.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:246]: use default wechat config => use the default wechat config

File: content/en/notification-manager/notification-manager.md [hunk 12]
```diff
@@ -0,0 +1,404 @@
...
+  toParty: global
+  toTag: global
+EOF
+```
+
+>Global wechat receiver use default wechat config.
+
+#### Slack
+
+Notification Manager supports sending notification to slack channels. If a tenant named test-user want to receive notification in slack, it must be create a slack receiver like this.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:304]: If a tenant named test-user want to =>If a tenant named test-user who wants to

Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:304]: it must be create => just create

File: content/en/notification-manager/notification-manager.md [hunk 13]
```diff
@@ -0,0 +1,404 @@
...
+    app: notification-manager
+  name: test-user-secret
+  namespace: kubesphere-monitoring-system
+type: Opaque
+EOF
+```
+
+>Slack token is the OAuth Access Token or Bot User OAuth Access Token when you create a slack app. This app must have the scope chat:write. The user who created the app or bot user must be in the channel which you want to send notification to.
+
+The slackConfigSelector is a selector to selecte SlackConfig for slack receiver, if the slackConfigSelector is nil, slack receiver will use default slack config. You can create a default slack config like this.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:355]: use default => use the default

File: content/en/notification-manager/notification-manager.md [hunk 14]
```diff
@@ -0,0 +1,404 @@
...
+    type: global
+  name: global-email-receiver
+  namespace: kubesphere-monitoring-system
+spec:
+  to:
+  - global@xyz.com
+EOF
+```
+
+>Global email receiver use default email config.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:183]: use default => use the default

File: content/en/notification-manager/notification-manager.md [hunk 15]
```diff
@@ -0,0 +1,404 @@
...
+  labels:
+    app: notification-manager
+    type: tenant
+    user: test-user
+spec:
+  channel: global
+EOF
+```
+
+>Global slack receiver use default slack config.
```
Comment (FeynmanZhou) [inline content/en/notification-manager/notification-manager.md:404]: use default => use the default

**Comment by wanjunlei** [comment]
This pr will be resubmit.

**Pull request closed without merging**